VotingApplication/VotingApplication.Web/Scripts/Controllers/ManageOptionController.js

```diff
@@ -0,0 +1,43 @@
+﻿(function () {
+    angular.module('GVA.Creation').controller('ManageOptionController', ['$scope', '$routeParams', '$location', 'ManageService', function ($scope, $routeParams, $location, ManageService) {
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.